controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaInterceptor.java

```diff
@@ -54,7 +54,7 @@
                 paramMap.put(key,
                         headers.get(Metadata.Key.of(key, Metadata.ASCII_STRING_MARSHALLER)));
             } catch (IllegalArgumentException e) {
-                log.warn("Error while marshalling some of the headers", e);
+                log.trace("Error while marshalling some of the headers", e);
```

Comment:
I am concerned that if we make this `trace`, we will never know that something happened. 

Would it make sense to do something different instead:
- Keep `log.warn`, but use `e.toString()` which will be a one-liner with just the exception type and message
- Add `log.trace` with `e`, which will output the stack trace, but only in trace mode.
    - We may even want to do this as `debug`.
    - Actually, is the stack trace even needed? If it's all internal to some external library, then all we'd need is the exception message and type, so no need for this second one.